analyses/oncoprint-landscape/01-plot-oncoprint.R

```diff
@@ -0,0 +1,210 @@
+# This script displays an oncoprint displaying the landscape across PBTA given 
+# the relevant metadata and output MAF files from the snv callers strelka2, 
+# mutect2, lancet, and vardict. It addresses issue #6 in the OpenPBTA-analysis
+# github repository.
+#
+# Chante Bethell for CCDL 2019 and Jo Lynne Rokita
+#
+# #### USAGE
+# This script is intended to be run via the command line from the top directory
+# of the repository as follows:
+#
+# Rscript 'analyses/oncoprint-landscape/01-plot-oncoprint.R'
+
+#### Set Up --------------------------------------------------------------------
+
+# Install maftools
+if (!("maftools" %in% installed.packages())) {
+  install.packages("BiocManager")
+  BiocManager::install("maftools")
+}
+library(maftools)
+
+# Define a color vector for plots 
+colores = c("Missense_Mutation" = "#35978f", 
+            "Nonsense_Mutation" = "#000000",
+            "Frame_Shift_Del" = "#56B4E9", 
+            "Frame_Shift_Ins" = "#FFBBFF", 
+            "Splice_Site" = "#F0E442",
+            "Translation_Start_Site" = "#191970",
+            "Nonstop_Mutation" = "#545454",
+            "In_Frame_Del" = "#CAE1FF",
+            "In_Frame_Ins" = "#FFE4E1",
+            "Stop_Codon_Ins" = "#CC79A7",
+            "Start_Codon_Del" = "#56B4E9",
+            "Fusion" = "#7B68EE",
+            "Multi_Hit" = "#f46d43",
+            "Hom_Deletion" = "#313695",
+            "Hem_Deletion" = "#abd9e9",
+            "Amplification" = "#c51b7d",
+            "Loss" = "#0072B2",
+            "Gain" = "#D55E00",
+            "High_Level_Gain" = "#FF0000",
+            "Multi_Hit_Fusion" = "#CD96CD")
+
+#### Functions -----------------------------------------------------------------
+
+# Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+
+plot_oncoplot <- function(maf_file, filename){
```

Comment:
Also think this is now a MAF object and not a MAF file.

Reply:
Good point, this change will be in the next commit.